xmodule/capa_block.py

```diff
@@ -74,24 +74,6 @@
     FEATURES = {}
 
 
-class SHOWANSWER:
-    """
-    Constants for when to show answer
-    """
-    ALWAYS = "always"
-    ANSWERED = "answered"
-    ATTEMPTED = "attempted"
-    CLOSED = "closed"
-    FINISHED = "finished"
-    CORRECT_OR_PAST_DUE = "correct_or_past_due"
-    PAST_DUE = "past_due"
-    NEVER = "never"
-    AFTER_SOME_NUMBER_OF_ATTEMPTS = "after_attempts"
-    AFTER_ALL_ATTEMPTS = "after_all_attempts"
-    AFTER_ALL_ATTEMPTS_OR_CORRECT = "after_all_attempts_or_correct"
-    ATTEMPTED_NO_PAST_DUE = "attempted_no_past_due"
-
-
```

Comment:
@steff456 `SHOWANSWER` class is being removed, but it is still referenced in `openedx/features/personalized_learner_schedules/show_answer/show_answer_field_override.py` and some tests.